app/src/main/java/mobi/inthepocket/android/beacons/app/utils/InputFilterMinMax.java

```diff
@@ -0,0 +1,44 @@
+package mobi.inthepocket.android.beacons.app.utils;
+
+import android.text.InputFilter;
+import android.text.Spanned;
+
+/**
+ * Created by elias on 23/10/2016.
+ */
+
+public class InputFilterMinMax implements InputFilter
+{
+    private int min, max;
```

Comment:
als ik me niet vergis, staat er ergens beschreven in onze coding guidelines dat we member variablen altijd apart definiëren